### ISSUE TITLE
Slide property active not updated 

### ISSUE THREAD

**Issue by pontusr** [valor-software/ngx-bootstrap#1267]
We're on ng2-bootstrap 1.1.16 and angular 2.1.0 but we've seen this issue for all combinations of releases.

We are showing a list of thumbnails of items contained in a carousel component. We depend on the active property on the slide items being updated when we call the select method on the carousel but this is does not happen.

```html
<span *ngFor="let image of thumbnailItems; let index = index"> 
<img  class="img-thumbnail" [ngClass]="{'showing': carouselItems[index].active}" (click)="switchSlide(index)" [src]="image.url" />
</span>
```
```typescript
  switchSlide(index) {
    for(let item of this.carouselItems)
    {
      item.active = false;
    }
    this.carouselItems[index].active = true;
}
```

**Comment by pontusr** [comment]
This issue would actually be solved by merging the commit already referenced in https://github.com/valor-software/ng2-bootstrap/issues/760

**Comment by IlyaSurmay** [comment]
Hi, could you reproduce this issue in plunkr/stackblitz?
You can use one of starter templates:
Plunkr: https://plnkr.co/edit/0NipkZrnckZZROAcnjzB?p=preview
StackBlitz: https://stackblitz.com/edit/ngx-bootstrap?file=app%2Fapp.module.ts

**Comment by IlyaSurmay** [comment]
Closed as can't reproduce (v2.0.0-beta.8), will be reopened if we have sample to reproduce
